app/supabase/functions/shared/lib/k8s-client.ts

```diff
@@ -230,7 +230,7 @@ export class K8sClient {
 
     try {
       console.log("Creating job with namespace:", namespace);
-      console.log("Job configuration:", JSON.stringify(job, null, 2));
+      console.log("Job configuration: [redacted env]");
```

Comment:
these make no sense to be logged without the details, they were just debug logs, only visible to operators. I'd rather remove them.